src/components/Resume/index.js

```diff
@@ -91,6 +95,7 @@ Resume.propTypes = {
   showExperience: PropTypes.bool,
   showCertification: PropTypes.bool,
   order: PropTypes.arrayOf(PropTypes.number),
+  paperSize: PropTypes.string,
```

Comment:
Invalid prop type.

Reply:
updated to be PropTypes.shape and changed the name from paperSize to paperSizeObj to avoid confusion with other instances of the paperSize variable which are simple strings